stackgl_modules/index.js

```diff
@@ -23536,6 +23536,19 @@ function makeVector(length, fill) {
   return result
 }
 
+// Fix for Chrome 128
+var INF = 1e38
+function ensureFinite(arr) {
+    return arr.map(function(v) {
+        return (
+            isNaN(v) ? 0 :
```

Comment:
Why does this one exclude NaN values? Doesn’t that pose a risk that missing data will be filled in spuriously?

Reply:
We didn't encounter that case in our tests.

Reply:
Do we have that case (missing data, particularly with lines but also points and surfaces) in our image tests? This is stackgl so I guess my question is scoped to the 3D trace types.

But also, this is different from what you did in parcoords, why?

Reply:
In stackgl jasmine tests I encountered a case with both infinity and NaN. So I had to guard against both.

Reply:
I still want to understand:
- Why are there differences between the stackgl and parcoords fixes? "because only one has a test that cares about NaN" isn’t enough. Have you tried to find a way to push a NaN into the other one? Does it cause problems? 
- Is there any chance that setting NaN to zero will make missing data suddenly appear? This I would consider a VERY bad bug, so it’s worth looking into carefully. Is there any way you can find to pass in an array containing some null data mixed in with valid data, or a typed array with NaN mixed in, that results in a point appearing where there should be none? If this code is processing the x/y/z arrays for scatter3D for example, I could imagine this happening due to NaN at the beginning, middle, or end of one of the arrays; NaN in all three at once; or different length arrays; all of these coming from either plain arrays or typed arrays. We could enumerate similar sets of conditions for the other trace types, and ideally we should have these conditions in test images so we have confidence the behavior is stable over time. These wouldn't need to all be different images, the same trace can have many incomplete or totally missing data points. And perhaps we already have some of these conditions represented in our test images - if so, fantastic, just point them out to me and we'll be good to go.

Reply:
The case of NaN didn't happen in the parcoords cases.
But it was present in a `mesh3d` case.
I start thinking perhaps NaNs could/should be avoided in one layer higher in the stackgl case.
Does that make sense?

Reply:
@archmoj @alexcjohnson is this a must-have for our next minor release?

Reply:
Given that several trace types seem fully broken on latest Chrome (and Edge, per https://github.com/plotly/plotly.py/issues/4729, not surprising as it’s built on Chrome) I certainly think so. I don’t particularly care how the fix is done but I do want us to test thoroughly that the fix can’t generate incorrect data points on a graph